code/obj/storage/vis_storage_controller.dm

```diff
@@ -0,0 +1,22 @@
+/datum/vis_storage_controller
+	var/obj/parent = null
+	///The items this storage controller contains
+	var/list/obj/vis_items = list()
```

Comment:
When declaring a variable that should initially be an empty list, it is better practice to assign `src.var_name = list()` in `New()`. This is because if a list is created in the variable's definition, BYOND will call a hidden `init()` proc to initialise the list. Creating the list in `New()` avoids the overhead of the `init()` call.